### DISCUSSION TITLE
Opinion: Could we get a more lively color than purple?

### DISCUSSION THREAD

**Discussion by UltraBurstXD** [ubuntu/yaru#2682] [Ideas]
Hello,
It may seem off-topic or radical to talk about the current color (Purple) chosen since 20.04, I think there was a great reason for such change. But for the sake of aesthetic could we get a more suitable color for both dark and light theme? I mean, a more lively color that attracts attention by it's beauty intensity?

**Comment by Feichtmeier** [comment]
Hi @UltraBurstXD

I also sometimes think that we should only use orange and green for suggested buttons
And then sparely use orange and avoid big orange areas like in the new row selection
But on the other hand purple sometimes looks very elegant, especially with the default theme

migrated this to discussions

CC @elioqoshi @madsrh

**Reply by UltraBurstXD** [reply]
Purple in light theme, may be elegant, but in dark theme it's awful. Then orange... I think there are already some people complaining about it's usage in certain areas. Orange, even though intuitive to certain extent, sometimes make things awkward quickly (or should I call it intrusive?), but I'd rather not talk about it.

I think we have orange enough, and it would spark indignation if Yaru goes "full" orange wannabe theme. Even applying sparely or slightly in remaining areas wouldn't bring satisfactory change.

Maybe colors like light cyan, aquamarine, magic mint or baby blue (even others variants of blue, green and cyan) are more elegant, sophisticated, attractive and vivid than purple.

**Reply by Feichtmeier** [reply]
I wouldn't call it awful but it lacks contrast. If you lighten the aubergine it becomes pink.
The Ubuntu orange is great if you use it in stripes and dots. Like cool black shoes with little orange accents

**Reply by elioqoshi** [reply]
Colour related discussions will always be a subjective to some extent, so we should try to be specific about what we think doesn't work and what work well, in order to make the case for other people to understand your argument. Calling a colour decision awful is not productive to a design process.

> Maybe colors like light cyan, aquamarine, magic mint or baby blue (even others variants of blue, green and cyan) are more elegant, sophisticated, attractive and vivid than purple.

I am not sure what that means. These are all personal interpretations based on the cultural background everyone perceives a colour in, but that's not universally applicable with that simplicitiy. I could argue as well that purple is a royal colour and hence is the most sophisticated. It doesn't add to the discussion though.

In any case, it would be good to have some references here in order to illustrate what you mean.
> If you lighten the aubergine it becomes pink.

We have a variety of possible tints and shades for Aubergine (which is referred to as purple here) and generally they are free to be used: https://design.ubuntu.com/brand/colour-palette/#tints

However using another colour which is not part of the Ubuntu brand family will probably not be possible here, unless there are high accessbility related issues with the colours (which we had with the orange sidebar navigation for example and resolved it).

**Comment by UltraBurstXD** [comment]
Maybe calling it awful was a bit too much... 

Aubergine or purple, wasn't so frequently used back on Ubuntu 19.10 like now. Green and Blue reigned in those times. Green was meh (simple and fresh)...

Canonical uses aubergine on their website, logo, and many others platforms or cloud services. It looks good there, but we can greatly note a enormous difference between Desktop (GTK theme) and website... It may look great there, but GTK gets a awkward lnf. A great color scheme choice can give GTK a refreshed look, without thinking too much, sometimes on HIG related issues.

It's possible to refresh GTK in web-lookalike way. But we all know that it's not easy and requires too much work. In programming world nothing is impossible.

Besides, I'm not saying that aubergine looks bad or something similar. I greatly appreciate it on terminal, lock screen, etc., but on some widgets... It's another thing. I only think, not suitable for some aspects of current Yaru GTK theme. Maybe others themes would be better suited.

**Reply by UltraBurstXD** [reply]
But in few words, I think sincerely the color chosen for some areas in 18.04 Communitheme, **blue**, for both light and dark theme was the wisest choice.

![34bd8-communitheme-gtk3-widget-factory](https://user-images.githubusercontent.com/55871338/109521617-2ba80400-7ab6-11eb-9403-5c52f4828a39.png)

**Reply by UltraBurstXD** [reply]
Maybe in material design guidelines convention, that Pop GTK Theme slightly uses would fair better... Roboto and Fira Sans fonts gives a great visual boost to text components:

![dash-to-dock](https://user-images.githubusercontent.com/55871338/109547138-b8ad8600-7ad3-11eb-9510-b2bf90d49fcb.png)

Yaru may attract many by it's simplicity and minimal usage of resources, attaining a highly optimized UX without lag because of some unnecessary effects. But, still an ideal color embellish even more the simple set of the components:

![maxresdefault](https://user-images.githubusercontent.com/55871338/109545179-2e642280-7ad1-11eb-94c8-6a386f7123e6.jpg)

I don't wanna talk about Pop OS GTK Theme vs Yaru vs Linux Mint... That's personal preference.

**Reply by elioqoshi** [reply]
Using typefaces or colours which are not part of the Ubuntu brand will likely not be an option. The colours used in Ubuntu 18.04 were not inline with the branding and hence were changed as well. On top of that various blue and green shades there lacked colour contrast and legibility.

For what it's worth, Pop uses bright colours which have low contrast and might not be an ideal choice accessibility-wise.

**Reply by UltraBurstXD** [reply]
Blue and Green weren't inline with the branding? I think I don't understand that because I still see green and blue in some areas. 

I don't think either that Sans Regular is part of Ubuntu brand (Well, it's open source, anyone can make it part of their brand...).

Color schemes is really a vague discussion... Ubuntu Budgie uses blue very well on their distro. They got another level of Simplicity and Elegance:
![budgie-remix](https://user-images.githubusercontent.com/55871338/109644798-a0cf1400-7b5e-11eb-9144-eb926afc2d41.png)

I really don't understand the so-found love for Aubergine. It messes up with many things using system lnf.

Why not bring a feature for users to customize some aspects of theme like Zorin OS:

![maxresdefault](https://user-images.githubusercontent.com/55871338/109642953-3f0daa80-7b5c-11eb-9d01-ee1e668c3790.jpg)

**Reply by UltraBurstXD** [reply]
Aren't Ubuntu Budgie core developers and designers using the same fonts or typefaces as Ubuntu? They are using Cairo renderer as well for what I know, so how can blue shades lack contrast and legibility?

If we where talking about skia and Cairo... But Cairo isn't so bad.

**Comment by UltraBurstXD** [comment]
I think I got a conclusion that some Yaru colors lacks contrast in some variants. And Dark theme can be cumbersome if we can't change to light theme afterwards. Well dark theme could work on night, but in daylight isn't very appealing. And maybe Canonical could consider adding more colors to their brand, to diversify some areas. 

Aubergine got little lively on Ubuntu 21.04, with little increase of contrast.

**Reply by Feichtmeier** [reply]
Buttons in the dark gtk theme have less contrast on the background than in 20.10 but still more than in adwaita. We'll look at in which direction we want to go there.
Adding more colours will probably not happen, and we also think about getting rid of the blue. Even if we love every single one of our colours, adding more than two is always problematic semanticly but most of all because in gtk is no theming API and we just follow Adwaita conventions.

**Reply by UltraBurstXD** [reply]
That's very troublesome, repetitive colors are very tiresome. Even Windows devs are trying some colors and there's less Blue.

And lack of contrast can be troublesome for some people with reduced eyesight.

**Reply by madsrh** [reply]
> That's very troublesome, repetitive colors are very tiresome...

That is very subjective. Many people complained that Yaru in 19.04 was waaaaaaay too colorful.
Anyway, as @Feichtmeier mentioned this is a GTK limitation, so there's not much we can do.

**Reply by elioqoshi** [reply]
> repetitive colors are very tiresome

That's not an objective and helpful sentiment unless you have some user study or other research reference to back this up with.

If there are contrast issues, we need to investigate it however.

**Reply by UltraBurstXD** [reply]
Hmm... Colorful?! How so? (Well to certain point... Yeah) If those guys could tell Google, Apple, Microsoft and Amazon that, I would appreciate it...

I think Canonical isn't communicative enough with community, maybe survey would help in this matter. Leaving that, if you could dig a bit, you'd see that many people found Ubuntu 19.04 and 19.10 very lively and vibrant. Many aspects on newest Yaru are way too dark shaded, it's like on few seconds everything would be dark.

I understand the GTK limitation, maybe QT would fair better in this case, but Linux doesn't have many reliable DE unless the interested party build it from scratch or fork one.
